xml-parser/src/main/java/org/uniprot/core/xml/dbreference/UniParcCrossReferenceConverter.java

```diff
@@ -0,0 +1,169 @@
+package org.uniprot.core.xml.dbreference;
+
+import com.google.common.base.Strings;
+import org.uniprot.core.uniparc.UniParcCrossReference;
+import org.uniprot.core.uniparc.UniParcDatabase;
+import org.uniprot.core.uniparc.impl.UniParcCrossReferenceBuilder;
+import org.uniprot.core.uniprotkb.taxonomy.Organism;
+import org.uniprot.core.uniprotkb.taxonomy.impl.OrganismBuilder;
+import org.uniprot.core.util.Utils;
+import org.uniprot.core.xml.Converter;
+import org.uniprot.core.xml.XmlReaderException;
+import org.uniprot.core.xml.jaxb.dbreference.DbReference;
+import org.uniprot.core.xml.jaxb.dbreference.ObjectFactory;
+import org.uniprot.core.xml.jaxb.dbreference.PropertyType;
+
+import org.uniprot.core.xml.uniprot.XmlConverterHelper;
+import org.uniprot.cv.taxonomy.TaxonomicNode;
+import org.uniprot.cv.taxonomy.TaxonomyRepo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class UniParcCrossReferenceConverter
```

Comment:
This class has many things similar with org.uniprot.core.xml.uniparc.UniParcDBCrossReferenceConverter. Do you think we could remove a bit of the duplicated code?